### PR TITLE
NXDOC-1354: Update readme

### PR DESCRIPTION
**Pull request by sguitter**
Fix and update links, update About Nuxeo part, update required Java version, remove obsolete extensions

### CODE REVIEW

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 1]
```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
...
 [18]: http://download.nuxeo.org/desktop-integration/drag-drop/msie/
-[19]: http://doc.nuxeo.com/x/K4Wo
-[21]: http://www.nuxeo.com/en/products/enterprise-platform/
-[22]: http://www.nuxeo.com/en/products/document-management
-[23]: http://www.nuxeo.com/en/products/dam
-[24]: http://www.nuxeo.com/en/products/case-management
-[25]: http://nuxeo.github.io/releasenotes/9.3-SNAPSHOT
-[26]: http://jira.nuxeo.org/browse/NXP?report=com.atlassian.jira.plugin.system.project:versions-panel
-[27]: https://jira.nuxeo.org/browse/NXP/fixforversion/
+[21]: http://www.nuxeo.com
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: https://www.nuxeo.com/

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 2]
```diff
@@ -113,14 +66,15 @@ with this release][27].
 
 ## Documentation and Help
 
-You will find documentation on the [the Nuxeo community site][28].
+You will find documentation on [doc.nuxeo.com][28].
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: https://doc.nuxeo.com/

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 3]
```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
 
 [1]: http://www.nuxeo.com/downloads/
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: https://www.nuxeo.com/downloads/

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 4]
```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
...
-[8]: http://doc.nuxeo.com/x/6QJc
+[5]: https://doc.nuxeo.com/nxdoc/adding-an-antivirus/
+[7]: https://doc.nuxeo.com/nxdoc/configuring-the-nuxeo-platform-as-a-daemon-with-sysvinit/
+[8]: https://doc.nuxeo.com/nxdoc/installing-the-nuxeo-platform-as-a-windows-service/
 [9]: http://localhost:8080/nuxeo/nxhome/default@view_admin?tabIds=NUXEO_ADMIN%3AConnectApps
 [10]: https://marketplace.nuxeo.com/
-[11]: https://doc.nuxeo.com/x/PwA7
+[11]: https://doc.nuxeo.com/nxdoc/administration/
 [17]: https://download.nuxeo.com/browser/firefox/nuxeo-dragdrop-ff-extension.xpi
 [18]: http://download.nuxeo.org/desktop-integration/drag-drop/msie/
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: not used anymore

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 5]
```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
...
-[7]: http://doc.nuxeo.com/x/NIpH
-[8]: http://doc.nuxeo.com/x/6QJc
+[5]: https://doc.nuxeo.com/nxdoc/adding-an-antivirus/
+[7]: https://doc.nuxeo.com/nxdoc/configuring-the-nuxeo-platform-as-a-daemon-with-sysvinit/
+[8]: https://doc.nuxeo.com/nxdoc/installing-the-nuxeo-platform-as-a-windows-service/
 [9]: http://localhost:8080/nuxeo/nxhome/default@view_admin?tabIds=NUXEO_ADMIN%3AConnectApps
 [10]: https://marketplace.nuxeo.com/
-[11]: https://doc.nuxeo.com/x/PwA7
+[11]: https://doc.nuxeo.com/nxdoc/administration/
 [17]: https://download.nuxeo.com/browser/firefox/nuxeo-dragdrop-ff-extension.xpi
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: not used anymore

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 6]
```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
...
-[21]: http://www.nuxeo.com/en/products/enterprise-platform/
-[22]: http://www.nuxeo.com/en/products/document-management
-[23]: http://www.nuxeo.com/en/products/dam
-[24]: http://www.nuxeo.com/en/products/case-management
-[25]: http://nuxeo.github.io/releasenotes/9.3-SNAPSHOT
-[26]: http://jira.nuxeo.org/browse/NXP?report=com.atlassian.jira.plugin.system.project:versions-panel
-[27]: https://jira.nuxeo.org/browse/NXP/fixforversion/
+[21]: http://www.nuxeo.com
+[25]: https://doc.nuxeo.com/nxdoc/nuxeo-server-release-notes/
+[26]: http://jira.nuxeo.com/browse/NXP?report=com.atlassian.jira.plugin.system.project:versions-panel
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: https://jira.nuxeo.com/browse/NXP

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt [hunk 7]
```diff
@@ -168,29 +122,25 @@ licenses donated by YourKit), etc.
...
+[27]: https://jira.nuxeo.com/browse/NXP/fixforversion/
 [28]: http://doc.nuxeo.com/
```
Comment (jcarsique) [inline nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: https://doc.nuxeo.com/

File: nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt (hunk 4 above)
Comment (sguitter) [reply nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/README.txt]: Actually there is still a page with various links.

**Comment by sguitter** [comment]
@jcarsique Files updated

**Comment by sguitter** [comment]
Too late for 9.3

**Pull request closed without merging**